server/migrations/20210609153902-add-threshold-to-create-topic.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        'OffchainTopics',
+        'token_threshold',
+        {
+          type: Sequelize.STRING,
+          allowNull: false,
```

Comment:
I feel like we could/should allow Topics to *not* have thresholds. In non-token communities for example.

Reply:
I was thinking `"0"` would do the job but you're right in a non-token community null makes more sense.